app/assets/javascripts/micropost_picture.js

```diff
@@ -0,0 +1,6 @@
+$("#micropost_picture").bind("change", function() {
```

Comment:
Without creating a new file, you can place this code in the application.js, as it can be reused by other file(picture) of the application.